.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Fix # .
+
+
+For significant contribution make sure you completed the list:
```

Comment:
nitpick: "For significant contributions please make sure you have completed the following items:"